<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1123</width>
    <height>655</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="CameraLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string>Camera 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RegisterButton">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>540</y>
      <width>160</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Myanmar Text</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Đăng ký mới</string>
    </property>
   </widget>
   <widget class="QPushButton" name="UpdateButton">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>540</y>
      <width>270</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Cập nhật người đăng ký mới</string>
    </property>
   </widget>
   <widget class="QLabel" name="HistoryLabel">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>290</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Lịch sử</string>
    </property>
   </widget>
   <widget class="QLabel" name="CameraShow">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="countinueButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>540</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Tiếp tục</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="Screen2">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>320</y>
      <width>160</width>
      <height>200</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Screen1">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>40</y>
      <width>440</width>
      <height>241</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="CameraLabel_2">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>10</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string>Camera 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AlertHasStranger">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>320</y>
      <width>270</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Cảnh báo có người lạ </string>
    </property>
   </widget>
   <widget class="QPushButton" name="AlertPeopleAtDoor">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>430</y>
      <width>270</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Có người trước cửa</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1123</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
